content/articles/two-factor-authentication-enforcement.markdown

```diff
@@ -0,0 +1,78 @@
+---
+title: Enforce Two-Factor Authentication for a shared Account
+excerpt: How to protect your shared DNSimple account using Two-Factor authentication enforcement.
+categories:
+- Account
+---
+
+# Enforce Two-Factor Authentication for a shared Account
+
+<info>
+This article describes a feature in Public Beta.
+</info>
+
+### Table of Contents {#toc}
+
+* TOC
+{:toc}
+
+---
+
+DNSimple supports [Two-Factor authentication](/articles/two-factor-authentication/). This feature enforces that all users in our account must have Two-Factor authentication enabled.
+
+When you have added other users to your account with it is important to ensure the security of each account member. This is why this feature is available to all plans that have access to the teams feature.
+
+## Enabling Two-Factor authentication enforcement {#enable}
+
+You can enable Two-Factor authentication enforcement from your account members page.
+
+You can only enable Two-Factor authentication enforcement for your account, if all users have Two-Factor authentication enabled. You either need to bug them to enable it or remove them from the team to enable this feature.
```

Comment:
Removing _your_ account. Users have multiple accounts, and they don't are the unique owners.

```suggestion
You can only enable Two-Factor authentication enforcement an account only if all users have Two-Factor authentication enabled. You either need to bug them to enable it or remove them from the team to enable this feature.
```

I think the following line may need some rewording:

> You either need to bug them to enable it or remove them from the team to enable this feature.